### PR TITLE
Extracting oas-to-har into @readme/oas-to-har

### PR DESCRIPTION
**Pull request by erunion**
* [x] Extracting [`oas-to-har.js`](https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/src/lib/oas-to-har.js) into a new @readme/oas-to-har package.
* [x] Replacing usages of [`Oas`](https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/src/lib/Oas.js) with the improved https://github.com/readmeio/oas module. See https://github.com/readmeio/oas/pull/62 for more information.
* [x] Adopting https://github.com/readmeio/eslint-config in the api-explorer package.

### CODE REVIEW

File: packages/api-explorer/src/Doc.jsx [hunk 1]
```diff
@@ -16,9 +18,7 @@ const Response = require('./Response');
 const ResponseSchema = require('./ResponseSchema');
 const EndpointErrorBoundary = require('./EndpointErrorBoundary');
 
-const Oas = require('./lib/Oas');
-const { Operation } = require('./lib/Oas');
-const getPath = require('./lib/get-path');
```
Comment (rafegoldberg) [inline packages/api-explorer/src/Doc.jsx]: Did you mean to delete this dependency? I see that `getPath` is still being referenced further down in this file... haven't fully traced so I very well may be wrong!

Comment (erunion) [reply packages/api-explorer/src/Doc.jsx]: Yep, `get-path` now exists in https://github.com/readmeio/oas and can be imported via `require('oas/utils').getPath`.

Comment (erunion) [reply packages/api-explorer/src/Doc.jsx]: See https://github.com/readmeio/oas/blob/master/utils.js

Comment (rafegoldberg) [reply packages/api-explorer/src/Doc.jsx]: perfect!

Comment (rafegoldberg) [reply packages/api-explorer/src/Doc.jsx]: merge away

**Pull request merged**